multitenancy-spring-boot-autoconfigure/src/test/java/org/axonframework/extensions/multitenancy/autoconfig/MultiTenantPersistentStreamMessageSourceTest.java

```diff
@@ -0,0 +1,145 @@
+package org.axonframework.extensions.multitenancy.autoconfig;
```

Comment:
If we move the `MultiTenantPersistentStreamMessageSource` to another directory, we should also move this file.